spec/src/main/asciidoc/spec.adoc

```diff
@@ -1209,64 +1209,3 @@ public class CalculatorBean {
     // ...
 }
 ----
-
-=== jakarta.annotation.ManagedBean
-
-The _ManagedBean_ annotation is used to
-declare a Jakarta Managed Bean as specified in the _Jakarta Managed Beans_
-specification. Jakarta Managed Beans are container-managed objects that support
-a small set of basic services such as resource injection, lifecycle
-callbacks and interceptors. A Jakarta Managed Bean may optionally have a name, a
-_String_ specified via the _value_ element.
-
-[source,java]
-----
-package jakarta.annotation;
-
-import static java.lang.annotation.ElementType.*;
-import static java.lang.annotation.RetentionPolicy.*;
-
-@Target(TYPE)
-@Retention(RUNTIME)
-public @interface ManagedBean {
-    boolean value() default "";
-}
-----
-
-|===
-|Element |Description |Default
-|value |Name of the Jakarta Managed Bean |""
-|===
-
-Examples:
-
-[source,java]
-----
-@ManagedBean("cart")
-public class ShoppingCart {
-    // ...
-}
-----
-
-== References
```

Comment:
I see no point in removing the references section, I'd rather see it being updated